modules/hawkular-api-parent/hawkular-listener/src/main/java/org/hawkular/listener/bus/InventoryEventListener.java

```diff
@@ -711,8 +707,7 @@ private void handleResourceEvent(ResourceEvent event) {
                     // Response Time
                     String groupTriggerId = "URL_Response";
                     String memberId = groupTriggerId + "_" + resourceId;
-                    String memberName = "URL Response [" + url + "]";
```

Comment:
It defaults to the group trigger name, which is currently the policy for all of the member triggers.